roles/dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2

```diff
@@ -0,0 +1,442 @@
+        username: wso2carbon
+        password: wso2carbon
+        driverClassName: org.h2.Driver
+        maxPoolSize: 50
+        idleTimeout: 60000
+        connectionTestQuery: SELECT 1
+        validationTimeout: 30000
+        isAutoCommit: false
+
+  - name: Message_Tracing_DB
+    description: "The datasource used for message tracer to store span information."
+    jndiConfig:
+      name: jdbc/Message_Tracing_DB
+    definition:
+      type: RDBMS
+      configuration:
+        jdbcUrl: '{{ message_tracing_db.url }}'
+        username: {{ message_tracing_db.username }}
+        password: {{ message_tracing_db.password }}
+        driverClassName: {{ message_tracing_db.driver_class_name }}
+        maxPoolSize: 50
+        idleTimeout: 60000
+        connectionTestQuery: SELECT 1
+        validationTimeout: 30000
+        isAutoCommit: false
+
+wso2.business.rules.manager:
+  datasource: BUSINESS_RULES_DB
+  # rule template wise configuration for deploying business rules
+  deployment_configs:
+    -
+     # <IP>:<HTTPS Port> of the Worker node
+     localhost:9443:
+       # UUIDs of rule templates that are needed to be deployed on the node
+       - stock-data-analysis
+       - stock-exchange-input
+       - stock-exchange-output
+       - identifying-continuous-production-decrease
+       - popular-tweets-analysis
+       - http-analytics-processing
+       - message-tracing-source-template
+       - message-tracing-app-template
+  # credentials for worker nodes
+  username: admin
+  password: admin
+
+wso2.status.dashboard:
+  pollingInterval: 5
+  metricsDatasourceName: 'WSO2_METRICS_DB'
+  dashboardDatasourceName: 'WSO2_STATUS_DASHBOARD_DB'
+  workerAccessCredentials:
+    username: '{{ worker.access_credentials.username }}'
+    password: '{{ worker.access_credentials.password }}'
+  queries:
+    -
+     mappings:
+        tableCreateQuery:
+        foreignKeyQuery:
+        tableCheckQuery:
+        recordSelectQuery:
+        recordSelectAppMetricsQuery:
+        recordSelectAgregatedAppMetricsQuery:
+        recordSelectWorkerThroughputQuery:
+        recordSelectWorkerAggregateThroughputQuery:
+        recordSelectWorkerMetricsQuery:
+        recordSelectWorkerAggregateMetricsQuery:
+        recordInsertQuery:
+        recordDeleteQuery:
+        selectAppComponentList:
+        selectAppComponentMetrics:
+        selectAppComponentHistory:
+        selectAppComponentAggregatedHistory:
+        doubleType:
+        floatType:
+        integerType:
+        longType:
+        stringType:
+        booleanType:
+
+     type: default
+     version: default
+
+wso2.transport.http:
+  transportProperties:
+    - name: "server.bootstrap.socket.timeout"
+      value: 60
+    - name: "client.bootstrap.socket.timeout"
+      value: 60
+    - name: "latency.metrics.enabled"
+      value: true
+
+  listenerConfigurations:
+    - id: "default-https"
+      host: "{{ wso2_carbon.host }}"
+      port: 9643
```

Comment:
Shouldn't we template `port` as well since can use different ports in a distributed environment?